script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -119,7 +119,7 @@ Tohno Shiki is to return to the Tohno household at once.'
 [sha:04b1459f61c4e8d1ad80da0e20b06ed713e52cc8]{
 -- Page 9, Offset 1169.
 -- 七年近く音沙汰のなかった実家からの電報は、そんな意味合いのものだったという。
-The telegram sent from my original household, whom I hadn't heard from in nearly seven years, said something to that effect.
+The telegram sent from my original household, from whom I hadn't heard from in nearly seven years, said something to that effect.
```

Comment:
double from feels a bit off to me
> **from** whom I hadn't heard **from**

Reply:
really? reads fine to me aloud though i guess if i was looking at it in a vacuum it might look unusual. Original reads quite awkwardly imo